include/deal.II/distributed/tria_base.h

```diff
@@ -202,21 +202,12 @@ namespace parallel
     const std::set<types::subdomain_id> &
     level_ghost_owners() const;
 
-    /**
-     * Return partitioner for the global indices of the cells on the active
-     * level of the triangulation, which is returned by the function
-     * CellAccessor::global_active_cell_index().
-     */
     const std::weak_ptr<const Utilities::MPI::Partitioner>
-    global_active_cell_index_partitioner() const;
+    global_active_cell_index_partitioner() const override;
```

Comment:
Aren't these functions the same as the base class ones now? I think we can just remove the implementation here.

Reply:
Hm. Lets deal with that later.